src/main/java/com/nessynet/yaacs/controller/AnnController.java

```diff
@@ -0,0 +1,46 @@
+package com.nessynet.yaacs.controller;
+
+import com.nessynet.yaacs.model.Anime;
+import com.nessynet.yaacs.model.ann.AnnAnime;
+import com.nessynet.yaacs.repository.AnimeRepository;
+import com.nessynet.yaacs.service.ann.AnnService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/ann")
+public class AnnController {
+	public static final Logger logger = LoggerFactory.getLogger(AnnController.class);
+
+	private AnnService annService;
+	private AnimeRepository animeRepository;
+
+	@Autowired
+	public AnnController(final AnnService annService, final AnimeRepository animeRepository) {
```

Comment:
I have reservations about making the controller have a hard dependency on the repository. Is this a standard approach?

In my mind we would have a "service" class responsible for tying the API that performs a rest call to Ann and the API that performs the db store together in a meaningful way.

Reply:
This will prevent db related exception handling in the controller...